### ISSUE TITLE
PC Sheet | Aspect Block internationalization

### ISSUE THREAD

**Issue by Oliver-Akins** [Oliver-Akins/foundry.dungeon#45]
Right now, the Aspect block in the PC sheet doesn't localize the labels, this should be updated to translate the labels appropriately.
